File: 12.09.2020/task 4.cpp
#include <iostream>

using namespace std;

int main() {
    int temp,a,n;
    bool flag = false;
    cin>>n;
    cin>> a;
    for (int i = 0; i < n; ++i)
    {
        cin>>temp;
        if (temp==a)
        {
            flag = true;
            //break;
        }
    }
    if (flag)
    {
        cout<<"Yes";
    }
    else
        {
        cout<<"No";
        }
}

File: Grade 10/25.11.2020/Task 4.pas
var a,check:integer;
flag:boolean;
begin
  read(a, check);
  flag := false;
  while(a>0) do 
  begin
    if (a mod 10=check) then
      flag := true;
    a := a div 10;
  end;
  if (flag) then
    write('Yes')
  else
    write('No');
end.